xbmc/interfaces/builtins/PlayerBuiltins.cpp

```diff
@@ -663,6 +685,9 @@ CBuiltins::CommandMap CPlayerBuiltins::GetOperations() const
            {"playercontrol",       {"Control the music or video player", 1, PlayerControl}},
            {"playmedia",           {"Play the specified media file (or playlist)", 1, PlayMedia}},
            {"playwith",            {"Play the selected item with the specified core", 1, PlayWith}},
-           {"seek",                {"Performs a seek in seconds on the current playing media file", 1, Seek}}
+           {"seek",                {"Performs a seek in seconds on the current playing media file", 1, Seek}},
+           {"subtitleshiftup",     {"Shift up the subtitle position", 0, SubtitleShiftUp}},
+           {"subtitleshiftdown",   {"Shift down the subtitle position", 0, SubtitleShiftDown}}
```

Comment:
You'll notice that an existing line is changed, dirtying git history. This can be avoided by using a trailing comma (which C++ allows for precisely for this reason)

Reply:
I did not understand what you are proposing here please provide example

Reply:
Change:

```c++
           {"subtitleshiftdown",   {"Shift down the subtitle position", 0, SubtitleShiftDown}}
```

to:

```c++
           {"subtitleshiftdown",   {"Shift down the subtitle position", 0, SubtitleShiftDown}},
```